test/client/domains.js

```diff
@@ -38,4 +44,47 @@ describe(`domains test`, () => {
             }
         }
     });
+
+    it('getPayPalLoggerDomain should return the logger domain when is a local environment', () => {
+        window.__ENV__ = ENV.LOCAL;
+        const result = getPayPalLoggerDomain();
+
+        if (result !== 'https://mock://msmaster.qa.paypal.com') {
+            throw new Error(`should get the logger domain "https://mock://msmaster.qa.paypal.com", but got: ${ result }`);
+        }
+    });
+
+    it('getPayPalLoggerDomain should thrown an Error when is a local environment and the stage host is undefined', () => {
+        const previousEnv =  window.__ENV__;
+        const previousStageHost =  window.__STAGE_HOST__;
+        window.__ENV__ = ENV.LOCAL;
+        window.__STAGE_HOST__ = undefined;
+        try {
+            getPayPalLoggerDomain();
+        } catch (err) {
+            if (err.message !== 'No stage host found') {
+                throw new Error(`should thrown exception with message "No stage host found", but got ${ err.message }`);
+            }
+        }
+        window.__ENV__ = previousEnv;
```

Comment:
This shouldn't be needed since we clean up `window.__ENV__` in the beforeEach().
```suggestion
```